examples/jaeger-hotrod/docker-compose.yml

```diff
@@ -1,36 +1,31 @@
-version: "3.7"
 services:
   data-prepper:
     restart: unless-stopped
-    container_name: data-prepper
     image: opensearchproject/data-prepper:2
     volumes:
-      - ../trace_analytics_no_ssl_2x.yml:/usr/share/data-prepper/pipelines/pipelines.yaml
+      - ./trace_analytics_no_ssl_2x.yml:/usr/share/data-prepper/pipelines/pipelines.yaml
       - ../data-prepper-config.yaml:/usr/share/data-prepper/config/data-prepper-config.yaml
-      - ../demo/root-ca.pem:/usr/share/data-prepper/root-ca.pem
+      - opensearch-config:/usr/share/opensearch-test/:ro
```

Comment:
Is this connecting the Docker volume running OpenSearch so that Data Prepper can get the certificate from there?

Reply:
Yes, I want to avoid updating the old `root-ca.pem` file by having read-only access. That old file is linked to multiple demos and is generated from `admin` user and the `admin` password. This file comes directly from the opensearch instance's `/usr/share/opensearch/config/root-ca.pem`. data-prepper specifies the location of the pem file through the yaml configuration.

Reply:
Thank you. This is a good improvement.